addons/hr_expense/models/hr_expense.py

```diff
@@ -75,10 +75,11 @@ def _get_employee_id_domain(self):
     total_amount = fields.Monetary("Total In Currency", compute='_compute_amount', store=True, currency_field='currency_id', tracking=True, readonly=False, inverse='_inverse_total_amount')
     untaxed_amount = fields.Monetary("Total Untaxed Amount In Currency", compute='_compute_amount_tax', store=True, currency_field='currency_id')
     company_currency_id = fields.Many2one('res.currency', string="Report Company Currency", related='company_id.currency_id', readonly=True)
-    total_amount_company = fields.Monetary('Total', compute='_compute_total_amount_company', store=True, currency_field='company_currency_id')
+    total_amount_company = fields.Monetary('Total', tracking=True,
+        compute='_compute_total_amount_company', inverse='_inverse_total_amount_company', store=True, currency_field='company_currency_id', readonly=False)
```

Comment:
@smetl might suggest something like this
```suggestion
    total_amount_company = fields.Monetary(
        string='Total',
        tracking=True,
        compute='_compute_total_amount_company', inverse='_inverse_total_amount_company', store=True,
        currency_field='company_currency_id',
        readonly=False)
```